code/modules/power/apc/apc_tool_act.dm

```diff
@@ -113,6 +113,12 @@
 		terminal.dismantle(user, W)
 		return TRUE
 
+/obj/machinery/power/apc/multitool_act(mob/living/user, obj/item/multitool/multi_tool)
+	. = ..()
+	if(panel_open && !opened)
+		wires.interact(user)
+		return TRUE
```

Comment:
``/obj/machinery/power/apc/attackby(obj/item/attacking_object, mob/living/user, params)`` has 
```
	if(panel_open && !opened && is_wire_tool(attacking_object))
		wires.interact(user)
		return
```
Why isn't it working there? We should check that instead of adding this

Reply:
`/obj/machinery/tool_act(mob/living/user, obj/item/tool, tool_type)
	if(SEND_SIGNAL(user, COMSIG_TRY_USE_MACHINE, src) & COMPONENT_CANT_USE_MACHINE_TOOLS)
		return TOOL_ACT_MELEE_CHAIN_BLOCKING	//hit for wirecutters
	. = ..()	//called for multitiool
	if(. & TOOL_ACT_SIGNAL_BLOCKING)
		return
	update_last_used(user)`
this function is returning early for wirecutters allowing it to continue down item/proc/melee_attack_chain
for multitool, it is continuing to call atoms/tool_act which blocks the apc/attackby call and tries to redirect it to multitool_act which doesn't exist.
this allows wirecutters to call apc/attackby and apc/wirecutter_act but multitool is only able to call apc/multitool_act
this is as far as i can get, not sure how to adjust this